core/src/com/mygdx/insanerpg/model/item/stackable/consumable/gem/WeaponGemSecondaryAttribute.java

```diff
@@ -0,0 +1,27 @@
+package com.mygdx.insanerpg.model.item.stackable.consumable.gem;
+
+import java.util.Random;
+
+public enum WeaponGemSecondaryAttribute {
+    PercentCritChance,PercentPhysicalDamage,PercentMagicalDamage,PercentStr,PercentInt,PercentDex,PercentVit,
+    PercentMoveSpeed,PercentLifesteal;
```

Comment:
Is lifesteal just applied on autos? or on all skills.

Reply:
added SpellVamp attribute as well